client/login/magic-login/emailed-login-link-successfully.jsx

```diff
@@ -17,18 +17,27 @@ import { login } from 'lib/paths';
 import Card from 'components/card';
 import RedirectWhenLoggedIn from 'components/redirect-when-logged-in';
 import { hideMagicLoginRequestForm } from 'state/login/magic-login/actions';
+import { getCurrentLocaleSlug } from 'state/selectors';
 import { recordPageViewWithClientId as recordPageView } from 'state/analytics/actions';
 import Gridicon from 'gridicons';
 
 class EmailedLoginLinkSuccessfully extends React.Component {
 	static propTypes = {
+		hideMagicLoginRequestForm: PropTypes.func.isRequired,
+		locale: PropTypes.string.isRequired,
 		recordPageView: PropTypes.func.isRequired,
 	};
 
+	componentDidMount() {
+		this.props.recordPageView( '/log-in/link', 'Login > Link > Emailed' );
+	}
+
 	onClickBackLink = event => {
 		event.preventDefault();
+
 		this.props.hideMagicLoginRequestForm();
-		page( login( { isNative: true } ) );
+
+		page( login( { isNative: true, locale: this.props.locale } ) );
```

Comment:
you don't need to manually redirect here, you already defined a `href` on your `a` tag.